vowpalwabbit/oaa.cc

```diff
@@ -34,7 +33,7 @@ void learn_randomized(oaa& o, VW::LEARNER::single_learner& base, example& ec)
   if (ld.label == 0 || (ld.label > o.k && ld.label != (uint32_t)-1))
     std::cout << "label " << ld.label << " is not in {1," << o.k << "} This won't work right." << std::endl;
 
-  ec.l.simple = {1., 0.f, 0.f};  // truth
+  ec.l.simple = {1., VW::UNUSED_1, VW::UNUSED_0};  // truth
```

Comment:
`UNUSED_0` for both?

Reply:
Yes this was deliberate.  One is weight (1) the other is cost (0).  Did not make sense to initialize weight to 0.  Although I get that this is not a strict replacement of the code before.